podaac/nose_tests_podaac/podaac_test.py

```diff
@@ -0,0 +1,83 @@
ND-GIGF5', u'PODAAC-PATHF-MON50', u'PODAAC-GHRAM-4FA01', u'PODAAC-AQR40-3W7DS', u'PODAAC-GHMWI-4FR04', u'PODAAC-J1SDR-NEGC0', u'PODAAC-TEDOT-DAT08', u'PODAAC-GHMWI-4FR01', u'PODAAC-GHMOY-4FE01', u'PODAAC-GHAMA-2PO02', u'PODAAC-TEOCN-EA005', u'PODAAC-MODAM-MO4N4', u'PODAAC-AQR40-3RQCS', u'PODAAC-GHVRS-2PO03', u'PODAAC-GHVRS-2PO02', u'PODAAC-TEOCN-0NJ05', u'PODAAC-ALTCY-TJ123', u'PODAAC-TELND-NC005', u'PODAAC-PATHF-DYD51', u'PODAAC-PATHF-DYD50', u'PODAAC-GHTMI-2PR04', u'PODAAC-AQR40-3W1AS', u'NODCNA-J2OPR-NC000', u'PODAAC-GH17G-2PN01', u'PODAAC-AQR40-3SVDS', u'PODAAC-MODSA-8D9N4', u'PODAAC-MODST-MO9N4', u'PODAAC-SSAT2-DAX01', u'PODAAC-GRJPL-L1B02', u'PODAAC-SASSX-L3UCD', u'PODAAC-J1JMR-ENH0C', u'PODAAC-SPUR1-MET00', u'PODAAC-AQR40-3DUCS', u'PODAAC-MODST-AN9N4', u'PODAAC-AQR40-3SUAS', u'NODCNA-J2IGD-NC000', u'PODAAC-QSBYU-GLBMF', u'PODAAC-AQR40-3WVAS', u'PODAAC-QSBYU-GLBML', u'PODAAC-REY3B-ERMON', u'PODAAC-AQR40-3RRAS', u'PODAAC-GHVRS-3UO24', u'PODAAC-GHMG2-2PO01', u'PODAAC-AQR40-3SCDS', u'PODAAC-AQR40-3S7AS', u'PODAAC-AQR40-3T7CS', u'PODAAC-QSBYU-ANBML', u'PODAAC-TEOCN-0GJ05', u'PODAAC-TE2DE-COE01', u'PODAAC-GHG13-3CO01', u'PODAAC-AQR40-3WCAS', u'PODAAC-AQR40-3SAAS', u'PODAAC-CCF30-01XXX', u'PODAAC-PATHF-ANN50', u'PODAAC-AQR40-3WUDS', u'PODAAC-PATHF-7DD50', u'PODAAC-SEABY-ANBIM', u'PODAAC-AQR40-3R7AS', u'PODAAC-PATHF-DYN51', u'PODAAC-ERS1B-SNEN0', u'PODAAC-GHG13-2PO01', u'PODAAC-GHG13-2PO02', u'PODAAC-MODST-1D4D4', u'PODAAC-TEOCN-ENJ05', u'PODAAC-AQR40-3R3DS', u'PODAAC-SPUR1-DRIFT', u'PODAAC-GHNMR-4FJ01', u'PODAAC-AQR40-3SVAS', u'PODAAC-RSX12-L2A01', u'PODAAC-GHMG3-2PO02', u'PODAAC-AQR40-3RUDS', u'PODAAC-SEAAM-L3X00', u'PODAAC-GMSLM-TJ123', u'PODAAC-SPUR1-SEASR', u'PODAAC-TEECO-OBPK8', u'PODAAC-GHN19-3CO02', u'PODAAC-GHRTN-4FJ01', u'NODCNA-J2IPR-NC000', u'PODAAC-AQR40-3WACS', u'PODAAC-GH17L-2PE01', u'PODAAC-QSXXX-L1B02', u'PODAAC-MODAM-AN9N4', u'PODAAC-AQR40-3R1CS', u'PODAAC-QSBYU-ARILT', u'PODAAC-MODSA-MO4D4', u'PODAAC-MODSA-MO9N4', u'PODAAC-TELND-GO005', u'PODAAC-GHSEV-2PE01', u'PODAAC-AQR40-3R1AS', u'PODAAC-AQR40-3S7DS', u'PODAAC-GHCMC-4FM02', u'PODAAC-AQR40-3SCCS', u'PODAAC-GRGAD-20G05', u'PODAAC-MODTM-1D4N4', u'PODAAC-QSBYU-ANBIM', u'PODAAC-MODTM-MO4N4', u'PODAAC-MODSA-AN9N4', u'PODAAC-NSBYU-SNEN0', u'PODAAC-AQR40-3WCDS', u'PODAAC-TEOCN-EGC05', u'PODAAC-REYN2-OIMOW', u'PODAAC-TPMGD-BIN0B', u'PODAAC-AQR40-3DCCS', u'PODAAC-GHNOY-4FE01', u'PODAAC-REYN2-OIMON', u'PODAAC-GRGAD-20J15', u'PODAAC-GHTMI-2GR01', u'PODAAC-TELND-GIJP5', u'PODAAC-NSJPL-L3I02', u'PODAAC-AQR40-3ZMCS', u'PODAAC-AQR40-3WUCS', u'PODAAC-SEABY-ARBML', u'PODAAC-GRGSM-20GA5', u'PODAAC-PATHF-ANN51', u'PODAAC-NAVOC-2FRAC', u'PODAAC-GH18L-2PN01', u'PODAAC-SEABY-ARBMF', u'PODAAC-GRGAB-20G05', u'PODAAC-SPUR1-CTD00', u'PODAAC-AQR40-3W7CS', u'PODAAC-QSBYU-ARBML', u'PODAAC-SEABY-ANBMF', u'PODAAC-QSBYU-ARBMF', u'PODAAC-SEABY-ANBML', u'PODAAC-TEOCN-ETG05', u'PODAAC-PATHF-5DN51', u'PODAAC-PATHF-5DN50', u'PODAAC-GRGSM-96C05', u'PODAAC-TPTMR-REP01', u'PODAAC-AQR40-3SVCS', u'PODAAC-TEOCN-ENG05']
```

Comment:
The exact available_granule_search_datasetIds should not be hard coded. In it's current form this test can and will break based upon the dataset id's being added to or reduced. This is an unstable test... always.
I would suggest that you make it more general.